doc/data.md

```diff
@@ -1,8 +1,15 @@
-Database
-========
+Data
+====
 
-Data model
-----------
+Raster
+------
+
+The maps displayed in report pages uses [NaturalEarth](http://www.naturalearthdata.com/) as their background layers. More specifically, the "Natural Earth II with Shaded Relief, Water, and Drainages" layer is used, in "medium size". Download the zip file from http://www.naturalearthdata.com/downloads/10m-raster-data/10m-natural-earth-2/.  The name of the zip is `NE2_LR_LC_SR_W_DR.zip`.
+
+Unzip the downloaded zip file in the `data` directory of the project. If you want to place it elsewhere you will need to adjust the `rasterfile` variable in the `development.ini` file. Note that `production.ini` assumes that the file `NE2_LR_LC_SR_W_DR` directory in under `/var/sig`, which is where the geo data is stored on the server.
```

Comment:
Note that `production.ini` assumes that the file is located in a `NE2_LR_LC_SR_W_DR` directory in under `/var/sig`